ykrt/build.rs

```diff
@@ -41,4 +45,18 @@ pub fn main() {
         .unwrap()
         .build()
         .unwrap();
+
+    // Build the gdb plugin.
+    env::set_current_dir("yk_gdb_plugin").unwrap();
+    let out = Command::new("make").output().unwrap();
```

Comment:
This might sound silly, but we'll probably notice the cost of this. Because all we're really doing is using it to check "is the output of `cc` older than the `.c` input" it's going to be *much* quicker to do this test ourselves in Rust code. I'm not going to worry about it for this PR, but at some point when you have a spare 10 minutes, maybe fix?

Reply:
I'm not sure what you mean. Which test?

Do you mean, don't invoke make, but call the compiler directly?

(I experimented with using the `cc` crate here, but it doesn't work well for this scenario because no rust code is actually linked to what we are building)

Reply:
Ah wait. I  you mean the timstamp check?

That's handled below by rerun-if-changed.

Reply:
Ok. We can optimise it later, but even with `make` in the mix, it's in the order of `0.05` seconds.

Reply:
Agreed.